draft-ietf-quic-transport.md

```diff
@@ -2187,7 +2190,10 @@ usable for this connection.  Failure to validate a path does not cause the
 connection to end unless there are no valid alternative paths available.
 
 An endpoint uses a new connection ID for probes sent from a new local address,
-see {{migration-linkability}} for further discussion.
+see {{migration-linkability}} for further discussion. An endpoint that uses
+a new local address should include a NEW_CONNECTION_ID frame in the probe,
```

Comment:
"SHOULD"? I don't think that we need to be this suggestive here, other than to note that lack of connection IDs will make probing - and responding to probes - impossible.

Reply:
Note that there is another issue in the same section. The text says that a packet containing PATH_CHALLENGE, PATH_RESPONSE, and PADDING frames is a "probing frame". Adding the NEW_CONNECTION_ID makes that a "non probing packet". I am not sure I fully get the consequences.

Reply:
The consequence is that you'd be initiating a change to the new path, and not just a probe.  I think we want to expand the definition of probing packets to include NEW_CONNECTION_ID as well.